### DISCUSSION TITLE
Need help about inheritance mapping polymorphic relationships

### DISCUSSION THREAD

**Discussion by bocanhcam** [symfony/symfony#48714] [Q&A]
![Screenshot from 2022-12-20 17-04-46](https://user-images.githubusercontent.com/35592955/208640475-9c556ab2-e147-4ffc-8753-540543f5e168.png)

I have 3 tables like the above.
I want to make the relationship between column Article with Customer and Member using ```creator_id``` and ```creator_type```.
I'm struggling with how to create relationships for these table entities.
I already use ```@ORM\DiscriminatorColumn(name="discriminator_type", type="string", length=255)``` to define the type of each entity so I don't think of another way how to do it.
Can anyone give me a sample code?
Thanks a lot!!!

**Comment by MatTheCat** [comment]
A discriminator is always found in one single table for a given inheritance hierarchy (see https://www.doctrine-project.org/projects/doctrine-orm/en/2.14/reference/inheritance-mapping.html) so you are doing something wrong. Could you show your entities class?
